### YamlMime:ManagedReference
items:
- uid: System.Printing.PrintingCanceledException
  id: PrintingCanceledException
  children:
  - System.Printing.PrintingCanceledException.#ctor
  - System.Printing.PrintingCanceledException.#ctor(System.String)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String)
  - System.Printing.PrintingCanceledException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Printing.PrintingCanceledException.#ctor(System.String,System.Exception)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.Exception)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32)
  - System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)
  langs:
  - csharp
  name: PrintingCanceledException
  nameWithType: PrintingCanceledException
  fullName: System.Printing.PrintingCanceledException
  type: Class
  summary: "Unntaket som oppstår når kode prøver å få tilgang til en avbrutt utskriftsjobben."
  remarks: "Hvis du oppretter en [!INCLUDE[TLA#tla_winclient](~/add/includes/ajax-current-ext-md.md)] program som inkluderer utskriftsfunksjonen og tillater jobbene annullere, må du håndterer dette unntaket. Et ubehandlet unntak av denne typen kan føre til en Windows presentasjon Foundation-program ikke svarer.      > [!CAUTION] > Klasser i den <xref:System.Printing>navneområdet støttes ikke for bruk i en Windows ASP.NET-program eller tjeneste.</xref:System.Printing> Forsøk på å bruke disse klassene fra i ett av disse programtyper kan føre til uventede problemer, for eksempel dårligere tjeneste ytelse og kjøretid unntak.       Hvis du vil skrive ut fra en Windows Forms-program, kan du se den <xref:System.Drawing.Printing>navneområde.</xref:System.Drawing.Printing>"
  syntax:
    content: 'public class PrintingCanceledException : System.Printing.PrintJobException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.Printing.PrintSystemException
  - System.Printing.PrintJobException
  implements: []
  inheritedMembers:
  - System.Printing.PrintJobException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Printing.PrintJobException.JobId
  - System.Printing.PrintJobException.JobName
  - System.Printing.PrintJobException.PrintQueueName
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor
  id: '#ctor'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException()
  nameWithType: PrintingCanceledException.PrintingCanceledException()
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException()
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klasse."
  syntax:
    content: public PrintingCanceledException ();
    parameters: []
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klasse som gir en bestemt feiltilstand."
  syntax:
    content: public PrintingCanceledException (string message);
    parameters:
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feiltilstanden.</xref:System.String>"
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String)
  id: '#ctor(System.Int32,System.String)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klasse som gir en feilkoden og feiltilstand."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkes som en HRESULT, en kodet numerisk verdi som er tilordnet til et bestemt unntak.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feiltilstanden.</xref:System.String>"
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(SerializationInfo,StreamingContext)
  nameWithType: PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klasse som gir bestemte <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> og <xref:System.Runtime.Serialization.StreamingContext>.</xref:System.Runtime.Serialization.StreamingContext> Denne konstruktøren, er beskyttet."
  syntax:
    content: protected PrintingCanceledException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Data som kreves for å serialisere eller deserialisering av et objekt."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Konteksten, inkludert kilde og mål for serialisert dataflyt."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String,Exception)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klasse som gir en bestemt feiltilstand, inkludert de underliggende føre til at av unntaket."
  syntax:
    content: public PrintingCanceledException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feiltilstanden.</xref:System.String>"
    - id: innerException
      type: System.Exception
      description: "Den underliggende feilen betingelse som forårsaket av <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref>."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.Exception)
  id: '#ctor(System.Int32,System.String,System.Exception)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klasse som gir en feilkoden og feiltilstand, inkludert de underliggende føre til at av unntaket."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message, Exception innerException);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkes som en HRESULT, en kodet numerisk verdi som er tilordnet til et bestemt unntak.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feiltilstanden.</xref:System.String>"
    - id: innerException
      type: System.Exception
      description: "Den underliggende feilen betingelse som forårsaket av <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref>."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32)
  id: '#ctor(System.Int32,System.String,System.String,System.String,System.Int32)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String,String,String,Int32)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klasse som gir en feilkoden feiltilstand, Skriv ut kønavn, jobbnavn og jobb-ID."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message, string printQueueName, string jobName, int jobId);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkes som en HRESULT, en kodet numerisk verdi som er tilordnet til et bestemt unntak.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feiltilstanden.</xref:System.String>"
    - id: printQueueName
      type: System.String
      description: "Navnet på køen vert utskriftsjobben som forårsaket unntaket."
    - id: jobName
      type: System.String
      description: "Navnet på utskriftsjobben som forårsaket unntaket."
    - id: jobId
      type: System.Int32
      description: "ID-nummeret utskriftsjobben som forårsaket unntaket."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)
  id: '#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)'
  parent: System.Printing.PrintingCanceledException
  langs:
  - csharp
  name: PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  type: Constructor
  assemblies:
  - ReachFramework
  namespace: System.Printing
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Printing.PrintingCanceledException&quot;> </xref> klassen med feilkoden, feiltilstand, skrive ut kønavn, jobbnavn, jobb-ID og underliggende årsaken til unntaket."
  syntax:
    content: public PrintingCanceledException (int errorCode, string message, string printQueueName, string jobName, int jobId, Exception innerException);
    parameters:
    - id: errorCode
      type: System.Int32
      description: "En <xref:System.Int32>som tolkes som en HRESULT, en kodet numerisk verdi som er tilordnet til et bestemt unntak.</xref:System.Int32>"
    - id: message
      type: System.String
      description: "En <xref:System.String>som beskriver feiltilstanden.</xref:System.String>"
    - id: printQueueName
      type: System.String
      description: "Navnet på køen vert utskriftsjobben som forårsaket unntaket."
    - id: jobName
      type: System.String
      description: "Navnet på utskriftsjobben som forårsaket unntaket."
    - id: jobId
      type: System.Int32
      description: "ID-nummeret utskriftsjobben som forårsaket unntaket."
    - id: innerException
      type: System.Exception
      description: "Den underliggende årsaken til unntaket."
  overload: System.Printing.PrintingCanceledException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Printing.PrintJobException
  isExternal: false
  name: System.Printing.PrintJobException
- uid: System.Printing.PrintingCanceledException.#ctor
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException()
  nameWithType: PrintingCanceledException.PrintingCanceledException()
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException()
- uid: System.Printing.PrintingCanceledException.#ctor(System.String)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Printing.PrintingCanceledException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(SerializationInfo,StreamingContext)
  nameWithType: PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Printing.PrintingCanceledException.#ctor(System.String,System.Exception)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.Exception)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,Exception)
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String,String,String,Int32)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32)
- uid: System.Printing.PrintingCanceledException.#ctor(System.Int32,System.String,System.String,System.String,System.Int32,System.Exception)
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  nameWithType: PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
  fullName: System.Printing.PrintingCanceledException.PrintingCanceledException(Int32,String,String,String,Int32,Exception)
- uid: System.Printing.PrintingCanceledException.#ctor*
  parent: System.Printing.PrintingCanceledException
  isExternal: false
  name: PrintingCanceledException
  nameWithType: PrintingCanceledException.PrintingCanceledException
